lib/wrappers/transform-node.ts

```diff
@@ -3,12 +3,14 @@ import rimraf from 'rimraf';
 import undefinedToNull from '../utils/undefined-to-null';
 import NodeWrapper from './node';
 import { TransformNodeInfo, CallbackObject } from 'broccoli-node-api';
+import OutputWrapper from './output';
 
+const FSMerger = require('fs-merger');
```

Comment:
We probably need to increase bus factor in this library before leveraging. Need at least a couple of folks that can merge PR's and publish npm releases.

I've also opened up [a number of issues](https://github.com/SparshithNR/fs-merger/issues) RE: the implementation in fs-merger over on its repo.